scripts/run-integration-tests.sh

```diff
@@ -83,12 +84,18 @@ TEST_IMAGE_VERSION=${IMAGE_VERSION:-$LOCAL_GIT_VERSION}
 : "${MANIFEST_CNI_VERSION:=master}"
 BASE_CONFIG_PATH="$DIR/../config/$MANIFEST_CNI_VERSION/aws-k8s-cni.yaml"
 TEST_CONFIG_PATH="$TEST_CONFIG_DIR/aws-k8s-cni.yaml"
+TEST_CALICO_PATH="$DIR/../config/$MANIFEST_CNI_VERSION/calico.yaml"
 
 if [[ ! -f "$BASE_CONFIG_PATH" ]]; then
     echo "$BASE_CONFIG_PATH DOES NOT exist. Set \$MANIFEST_CNI_VERSION to an existing directory in ./config/"
     exit
 fi
 
+if [[ ! -f "$TEST_CALICO_PATH" && $RUN_CALICO_TEST == true ]]; then
```

Comment:
nit: we should check $`RUN_CALICO_TEST == true` before checking if `! -f "$TEST_CALICO_PATH"` because if the test is diabled and the file is not existing, we still echo the error message.

Reply:
But the file should be always available but yeah if the test is disabled we don't even have to check the file. Will update it.